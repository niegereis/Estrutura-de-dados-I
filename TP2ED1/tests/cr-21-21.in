21 21
r
*********************
*               *   *
* *********** *** * *
*   *       * *   *M*
*** * ******* * *****
* *   *       *   * *
* *** * ***** *** * *
*     * *   *   *   *
* ***** * * ******* *
*   *   * *   *     *
*** * * * *** * *****
* * * * * * * * *   *
* * * * * * * * * * *
* * * * * * *   * * *
* * * * * * ******* *
* * * * *     *     *
* * * * ***** * *** *
* * * * *   * *   * *
* * * *** * * *** * *
*   *     *       *  
*********************
