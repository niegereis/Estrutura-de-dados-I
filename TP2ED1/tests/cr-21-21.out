73
*********************
*               *ooo*
* *********** ***o*o*
*   *       * *ooo*M*
*** * ******* *o*****
* *   *       *ooo* *
* *** * ***** ***o* *
*     * *ooo*   *ooo*
* ***** *o*o*******o*
*   *   *o*ooo*ooooo*
*** * * *o***o*o*****
* * * * *o* *o*o*   *
* * * * *o* *o*o* * *
* * * * *o* *ooo* * *
* * * * *o* ******* *
* * * * *ooooo*ooooo*
* * * * *****o*o***o*
* * * * *   *o*ooo*o*
* * * *** * *o***o*o*
*   *     *  ooooo*oo
*********************
